### ISSUE TITLE
Moonshine Hangs After Restart (TODO:  update title after cause determined)

### ISSUE THREAD

**Issue by JoelProminic** [prominic/Moonshine-IDE#25]
I have seen this behavior twice in the last couple of days:
1. Restart Moonshine
2. Moonshine opens
3. Moonshine expands to full screen (because of my settings)
4. After about 5-10 seconds Moonshine hangs and needs to be killed
5. This issue repeats until you clear the configuration (~/Library/Containers/com.moonshine-ide)

The hang happens before the console window expands, and the timing makes me thing that the issue is related to this.

Unfortunately, I have not been able to reproduce the action from the Moonshine UI.  

I saved the configuration files from when the application was failing.  After some experimentation, I was able to reproduce the crash by replacing only this file:
~/Libraries/Containers/com.moonshine-ide/Data/Library/Preferences/Macromedia/Flash Player/macromedia.com/support/flashplayer/sys/settings.sol

Corrupt version (zipped so that I could attach it on GitHub):
[2017_09_22__Bad_moonshine-ide-local.sol.zip](https://github.com/prominic/Moonshine-IDE/files/1325888/2017_09_22__Bad_moonshine-ide-local.sol.zip)

Note that this is a binary file, but you can see some of the values if you open it with a text editor.  Since it has a lot of references to paths on my workstation, you probably won't be able to test with it directly, but I hope it will give some clues about what causes the hang.

**Comment by rat-moonshine** [comment]
After a trail of investigation, I found an weird behavior in OSX Moonshine. 

Unlike Windows, if the console panel set it's height more than 267 pixels - Moonshine hangs. I still not able to detect where its failing and make the application unusable, but after setting the height it shows app freeze.

**Comment by rat-moonshine** [comment]
I seen we had the concept to open the project SWF (in case with **Build & Run** for browser based projects) with their corresponding wrapper HTML file. We have the related wrapper HTML wrapper template file also, supplied within the project templates, for this purpose. I can vaguely remember we worked on this wrapper html concept to overcome this `navigateToURL` issue only (probably in OSX mainly).

When projects are created its bin-debug folder populates with the wrapper HTML files as well. But I noticed the current source continue to open .swf file instead of the .html in bin-debug folder, I don't remember when or why we make this changes.

Opening the wrapper .html file with `navigateToURL` seems solves the problem in OSX sandbox. 

However, there is other test case which needs to work on. When user do a **Clean Project** call, all the files inside the bin-debug folder deletes, including the wrapper HTML files. 

Those files never gets back again when you compile the project. We need to restore those template files again after a clean project call, during compilation. I'm working on it.
